docs/api/swagger.json

```diff
@@ -1,1499 +1,2124 @@
-                        "$ref": "#/components/responses/404"
-                    }
-                }
-            }
+        "responses": {
+          "204": {
+            "$ref": "#/components/responses/204"
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        }
+      }
+    },
+    "/import‎": {
+      "post": {
+        "summary": "Import group",
+        "tags": ["group"],
+        "operationId": "importGroup",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/active"
+          },
+          {
+            "$ref": "#/components/parameters/activate"
+          },
+          {
+            "$ref": "#/components/parameters/languagePackageId"
+          }
+        ],
+        "requestBody": {
+          "$ref": "#/components/requestBodies/ImportGroup"
         },
-        "/group/{groupId}/vocabulary": {
-            "get": {
-                "summary": "Get all vocabulary belonging to specific group",
-                "tags": [
-                    "vocabulary"
-                ],
-                "operationId": "getGroupVocabulary",
-                "parameters": [{
-                    "$ref": "#/components/parameters/groupId"
-                }],
-                "responses": {
-                    "200": {
-                        "$ref": "#/components/responses/GetGroupVocabulary"
-                    },
-                    "400": {
-                        "$ref": "#/components/responses/400"
-                    },
-                    "401": {
-                        "$ref": "#/components/responses/401"
-                    },
-                    "404": {
-                        "$ref": "#/components/responses/404"
-                    }
-                }
-            }
+        "responses": {
+          "204": {
+            "$ref": "#/components/responses/204"
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        }
+      }
+    },
+    "/inviteCode": {
+      "get": {
+        "summary": "Get all invite codes. 'null' means no restrictions regarding this property",
+        "tags": ["invite code"],
+        "operationId": "getInviteCodes",
+        "security": [
+          {
+            "bearerAuthAdmin": []
+          }
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/components/responses/GetInviteCodes"
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
```

Comment:
no 404 but 403 (should also be changed on the other invite resources, afaik `POST /inviteCode` is the only which can possibly return 400)